sherpa/astro/tests/test_astro_data_notebook.py

```diff
@@ -67,7 +62,7 @@ def check(r, summary, name, label, nmeta):
     assert '<div class="dataval">{}</div>'.format(name) in r
 
 
-@pytest.mark.parametrize('header', [None, TEST_HEADER])
+@pytest.mark.parametrize('header', [None, {}, TEST_HEADER])
```

Comment:
We should be able to drop the None case here (unless we think there is code out in sherpa that does end up setting header to None)?

Reply:
Not in sherpa itself, but users may have written code for `dat = DataPHA(..., header=None)` so I want to make sure that still works.
I think we could remove that, but in that case we should issue a DeprecationWarning first for at least one full version, and there is no harm done keeping it around.